src/pages/Patch.tsx

```diff
@@ -51,3 +65,14 @@ export const Patch = () => {
     </PageWrapper>
   );
 };
+
+const BadgeWrapper = styled.span`
+  display: inline-flex;
+`;
+
+const mapPatchStatusToBadgeVariant = {
```

Comment:
super nit: i wouldn't include map in the variable name

Reply:
i would